cmd/state-tray/pkg/autostart/autostart.go

```diff
@@ -5,13 +5,14 @@ import (
 	"path/filepath"
 	"runtime"
 
+	"github.com/ActiveState/cli/internal/config"
 	"github.com/ActiveState/cli/internal/osutils/autostart"
 )
 
-func New() *autostart.App {
+func New(cfg *config.Instance) *autostart.App {
 	suffix := ""
 	if runtime.GOOS == "windows" {
 		suffix = ".exe"
 	}
-	return autostart.New("activestate-desktop", filepath.Join(filepath.Dir(os.Args[0]), "state-tray"+suffix))
+	return autostart.New("activestate-desktop", filepath.Join(filepath.Dir(os.Args[0]), "state-tray"+suffix), cfg)
```

Comment:
I realize this isn't part of this PR but you might as well update this to `appinfo.TrayApp().Exec()`